### DISCUSSION TITLE
Referente ao curso Modelando domínios ricos

### DISCUSSION THREAD

**Discussion by marcelocaselli** [balta-io/forum#407] [Tira Dúvidas] [answered]
Balta bom dia,
No tópico organizando a solução, a forma que você cria as pastas e depois add classlib e etc, essa seria a abordagem DDD?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marcelocaselli, como vai?
Não necessariamente... uma classlib é apenas um projeto que no fim gera uma biblioteca (DLL).
DDD é um conjunto de conceitos que ajuda na modelagem do software.
